saas/models/saas_template.py

```diff
@@ -221,21 +222,22 @@ def create_db(self, key_values=None, db_name=None, with_delay=True, draft_build_
 
         self.env['saas.log'].log_db_creating(build, self.operator_db_id)
         if with_delay:
-            build.with_delay().create_db(
+            job_create=(build.with_delay().create_db(
                 self.operator_db_name,
                 self.template_id.template_demo,
-            )
-            build.with_delay().action_install_missing_mandatory_modules()
+            ))
+            job_missing_mandatory=(group(job_create).on_done(
```

Comment:
job_missing_mandatory is confusing, I would call it job_install_modules to comply with idea "job_<what this job does>"

Reply:
@em230418 changed